docs/index.rst

```diff
@@ -19,35 +18,36 @@ You can interact with the service in two ways:
 - or locate yourself, based on the radio environment around you.
 
 
-About the name
-==============
-
-In Greek mythology, Ichnaea (Iknaia) means "the tracker".
-
-
 Table of contents
 =================
 
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
 
-   usage
-   local_dev
-   api/index
-   import_export
-   install/index
+   user_index
+   devdeploy_index
    algo/index
-   glossary
    changes/index
+   glossary
+
+
+Indices
+=======
+
+* :ref:`genindex`
+* :ref:`glossary`
 
 
-Development
-===========
+Source code and license
+=======================
 
-All source code is available on `github under ichnaea
+All source code is available on `GitHub under ichnaea
 <https://github.com/mozilla/ichnaea>`_.
 
-License
-=======
+The ``Ichnaea`` source code is offered under the Apache License 2.0.
+
 
-The ``ichnaea`` source code is offered under the Apache License 2.0.
+About the name
+==============
+
+In Greek mythology, Ichnaea (Iknaia) means "the tracker".
```

Comment:
I'm glad this detail survived, and this is a better place for it. We could throw in the Greek (Ιχναίη from https://en.wikipedia.org/wiki/Ichnaea) and get someone to pronounce it for us at the next All Hands.